conda_forge_tick/mamba_solver.py

```diff
@@ -296,7 +296,7 @@ def _is_recipe_solvable_on_platform(recipe_dir, cbc_path, platform, arch):
     # here we extract the conda build config in roughly the same way that
     # it would be used in a real build
     config = conda_build.config.get_or_merge_config(
-        None, exclusive_config_file=cbc_path, platform=platform, arch=arch,
+        None, platform=platform, arch=arch, variant_config_files=[cbc_path],
```

Comment:
@wolfv IDK if you have used this code anywhere, but I found this bug in the order the config files are applied. In a real build, the .ci_support files generated by conda-smithy come last. If you have not used this bit of code anywhere else, then no need to worry about this.